### PR TITLE
Model inspection (data loading, processing, augmentation and normalization)

### PR DESCRIPTION
**Pull request by Jentesomers**

### CODE REVIEW

File: custom.py [hunk 1]
```diff
@@ -56,18 +58,22 @@ def load_custom(self, dataset_dir, subset):
...
         # Train or validation dataset?
         assert subset in ["train", "val"]
         dataset_dir = os.path.join(dataset_dir, subset)
 
         # We mostly care about the x and y coordinates of each region
-        annotations1 = json.load(open('D:\\env_with_tensorflow1.14\\all_maskrcnn\\maskrcnn_truck_car\\Dataset\\train\\train_json.json'))
-        # print(annotations1)
-        annotations = list(annotations1.values())  # don't need the dict keys
+        annotations = json.load(open(os.path.join(dataset_dir,
+                                 'via_region_data.json')))
```
Comment (VerleysenNiels) [inline custom.py:76]: Is dit de naam van je json file met annotaties?

Comment (Jentesomers) [reply custom.py:76]: ja, in elke folder staat nu een json file met als naam 'via_region_data'

File: mrcnn/proposal_layer.py [hunk 2]
```diff
@@ -13,7 +13,7 @@
 from tensorflow.python.eager import context
 import tensorflow.keras.models as models
 
-import utils
+from mrcnn import utils
```
Comment (VerleysenNiels) [inline mrcnn/proposal_layer.py:16]: Zeker dat dit werkt? Zijn die oude utils hetzelfde als de mrcnn.utils?

**Review by VerleysenNiels — APPROVED** [review]
Ziet er goed uit
